### ISSUE TITLE
Convert to JSX/ES6 to encourage contributions

### ISSUE THREAD

**Issue by irskep** [literallycanvas/literallycanvas#278]
Almost no one besides the core contributors seem to be familiar with Coffeescript. With the advent of ES6 transpilers, JavaScript is a more reasonable language than it was when we began.

I'm not yet sure how or when this transformation will occur.

**Comment by DexterHaslem** [comment]
I see this is really old but is it something you are still interested in doing at some point? I'd consider it if so. Mostly to become familiar with core so I could help with maintenance

**Comment by irskep** [comment]
Yep, I'd still like it to happen, but I also won't do it myself.

I recommend a file-by-file conversion rather than trying to do the whole project at once. You might need to refactor the build system (again). Or not. I'm not sure.

**Comment by DexterHaslem** [comment]
Ok thanks, I may start giving it a stab in small chunks once I fully understand the build system
